android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -102,8 +103,26 @@ class RNPlayerView(
     private val activityLifecycle = (context.currentActivity as? ReactActivity)?.lifecycle
         ?: error("Trying to create an instance of ${this::class.simpleName} while not attached to a ReactActivity")
 
+    /**
+     * Relays the provided set of events, emitted by the player, together with the associated name
+     * to the `eventOutput` callback.
+     */
+    private var playerEventRelay: EventRelay<Player, Event>
```

Comment:
You can just initialize the eventRelay here, no need for an extra init block 🙂

Reply:
Address in 503a4fc2cad5854751751937e115b988c9f53646